### PR TITLE
Parameter updates torso

### PR DESCRIPTION
**Pull request by fmessmer**
related to https://github.com/ipa320/cob_control/pull/87

### CODE REVIEW

File: cob_hardware_config/cob4-2/config/torso_driver.yaml [hunk 1]
```diff
@@ -2,24 +2,28 @@ defaults:
...
   - name: torso_2_joint
     id: 32
+    #publish: ["60C1sub1"]   #trajectory
+    #publish: ["60FF"]       #velocity
     dcf_overlay: # "ObjectID": "ParameterValue" (both as strings)
-      "6083": "6000" # profile acceleration, mgrad/sec^2
-      "6084": "6000" # profile deceleration, mgrad/sec^2
-      "60C5": "6000" # max acceleration, mgrad/sec^2
-      "60C6": "6000" # max deceleration, mgrad/sec^2
+      "6083": "200000" # profile acceleration, mgrad/sec^2
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/torso_driver.yaml:16]: that's a enormous increase. Are you satisfied with the behaviour?

File: cob_hardware_config/cob4-2/config/torso_driver.yaml [hunk 2]
```diff
@@ -2,24 +2,28 @@ defaults:
...
   - name: torso_3_joint
     id: 33
+    #publish: ["60C1sub1"]   #trajectory
+    #publish: ["60FF"]       #velocity
     dcf_overlay: # "ObjectID": "ParameterValue" (both as strings)
-      "6083": "80000" # profile acceleration, mgrad/sec^2
-      "6084": "80000" # profile deceleration, mgrad/sec^2
-      "60C5": "80000" # max acceleration, mgrad/sec^2
-      "60C6": "80000" # max deceleration, mgrad/sec^2
+      "6083": "200000" # profile acceleration, mgrad/sec^2
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/torso_driver.yaml:25]: that's a huge increase. Are you satisfied with the behaviour?

**Comment by floweisshardt** [comment]
CR: OK

**Comment by NadiaHG** [comment]
FR: OK

**Pull request merged**